### PR TITLE
Add test to ensure important metrics are available in prometheus.

### PR DESCRIPTION
**Pull request by sayanh**
**Description**

Changes proposed in this pull request:

- Add test to validate all required metrics are available in promethues.

**Related issue(s)**
#2541

### CODE REVIEW

File: tests/test-logging-monitoring/test-lambda-dashboard-requirements.go [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+		resp, err := http.Get(u)
+		if err != nil {
+			return fmt.Errorf("Call to prometheus failed: %v", err)
+		}
+
+		defer resp.Body.Close()
+		bodyBytes, _ := ioutil.ReadAll(resp.Body)
+		json.Unmarshal(bodyBytes, &s)
+
+		if s.Status != "success" {
```
Comment (rakesh-garimella) [inline tests/test-logging-monitoring/test-lambda-dashboard-requirements.go]: may be printing the response code  also would be useful.

File: tests/test-logging-monitoring/test-lambda-dashboard-requirements.go [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+			return fmt.Errorf("Call to prometheus failed with response: %v", s)
+		}
+
+		if len(s.Data) < 1 {
+			return fmt.Errorf("Metric or Lable not found: %s, %s", l, metric)
```
Comment (rakesh-garimella) [inline tests/test-logging-monitoring/test-lambda-dashboard-requirements.go]: typo: `label` also the order is wrong it should be: `return fmt.Errorf("Metric or Lable not found: %s, %s", metric, l)`

Comment (sayanh) [reply tests/test-logging-monitoring/test-lambda-dashboard-requirements.go]: fixed

File: tests/test-logging-monitoring/test-lambda-dashboard-requirements.go (hunk 1 above)
Comment (sayanh) [reply tests/test-logging-monitoring/test-lambda-dashboard-requirements.go]: fixed

**Pull request merged**